packages/x-license-pro/src/generateLicense/generateLicense.ts

```diff
@@ -1,30 +1,35 @@
 import { md5 } from '../encoding/md5';
 import { base64Encode } from '../encoding/base64';
+import { LICENSE_SCOPES, LicenseScope } from '../utils/licenseScope';
+import { LICENSING_MODELS, LicensingModel } from '../utils/licensingModel';
 
 const licenseVersion = '2';
 
-export type LicenseScope = 'pro' | 'premium';
-
 export interface LicenseDetails {
   orderNumber: string;
   expiryDate: Date;
   // TODO: to be made required once the store is updated
   scope?: LicenseScope;
+  // TODO: to be made required once the store is updated
+  licensingModel?: LicensingModel;
 }
 
 function getClearLicenseString(details: LicenseDetails) {
-  return `ORDER:${
-    details.orderNumber
-  },EXPIRY=${details.expiryDate.getTime()},KEYVERSION=${licenseVersion},SCOPE=${details.scope}`;
+  if (details.scope && !LICENSE_SCOPES.includes(details.scope)) {
+    throw new Error('MUI: Invalid scope');
+  }
+
+  if (details.licensingModel && !LICENSING_MODELS.includes(details.licensingModel)) {
+    throw new Error('MUI: Invalid sales model');
+  }
+
+  return `O=${details.orderNumber},E=${details.expiryDate.getTime()},S=${
```

Comment:
Without this change, a v2 license key is not usable by < v5.11.0 of the software. Why not keep?

```suggestion
  return `O=${details.orderNumber},EXPIRY=${details.expiryDate.getTime()},S=${
```

it was surfaced in https://groups.google.com/a/mui.com/g/x/c/Nkq2fQbGkn0/. Time will tell if this causes trouble with more users, I suspect it will. If the intent is to shorten the string key, I doubt it's worth the breaking change. If a breaking change needs to happen,  then I think bundling with #4892 could be great.

Reply:
Why revert instead of just supporting both in the decoding script?

Reply:
@joserodolfofreitas we are one week in, how are the upgrades support cases going? I haven't seen new ones, so maybe it's fine.